dispatch-client/src/main/scala/org/htt4s/client/dispatch/DispatchClient.scala

```diff
@@ -0,0 +1,50 @@
+package org.htt4s.client.dispatch
+
+import dispatch._, Defaults._
+import com.ning.http.client.{Response => DResponse}
+
+import org.http4s._
+import org.http4s.client._
+
+import scala.collection.JavaConverters._
+import scalaz.concurrent.Task
+
+import scalaz._
+import Scalaz._
+
+object DispatchClient {
+  def apply(): Client = {
+    Client(Service.lift { req =>
+      Task.async[DResponse] { register =>
+        Http(toDRequest(req)).onComplete {
+          case scala.util.Success(res) => register(res.right)
+          case scala.util.Failure(ex) => register(ex.left)
+        }
+      }.flatMap(fromDResponse)
+    }, Task.now(()))
+  }
+
+  private def toDRequest(request: Request): Req = {
+    url(request.uri.toString)
+      .setMethod(request.method.toString)
+      .setHeaders(request.headers
+        .groupBy(_.name.toString)
+        .mapValues(_.map(_.value).toSeq)
+      )
+      .setBody(request.bodyAsText.runFoldMap(identity).run)
```

Comment:
This is only going to work for text, and will materialize the entire request body.  Something like this would stream the bytes:

```
.setBody(new InputStreamBodyGenerator(scalaz.stream.io.toInputStream(request.body))
```